src/components/pages/gfcrPages/GfcrIndicatorSetForm/subPages/FinanceSolutions.js

```diff
@@ -101,7 +101,7 @@ const FinanceSolutions = ({ indicatorSet, setIndicatorSet, choices, displayHelp
 
       const sectorName = choices.sectors.data?.find(
         (sectorChoice) => sectorChoice.id === sector,
-      ).name
+      )?.name
```

Comment:
Typescript, bla bla :)

Reply:
This is the only thing that has been added in this PR. I like it